portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Task {
+
+  private final long id;
+  private final String name;
+  private final String comments;
+  private final long timestamp;
+
+  public Task(long id, String name, String comments, long timestamp) {
+    this.id = id;
+    this.name = name;
+    this.comments = comments;
+    this.timestamp = timestamp;
+  }
+}
```

Comment:
Not a huge deal, but it's generally good practice to add a newline at the end of each file. Our code linters will actually block you from submitting if you forget to add one, and it looks like Github shows a little warning icon too. There's a couple other files below that are also missing newlines at the end.